code-list-scripts/cypher/ott_countries_in_groups.cypher

```diff
@@ -0,0 +1,141 @@
+CREATE CONSTRAINT ON (n:`_code_ott_countries_in_groups`) ASSERT n.code IS UNIQUE;
```

Comment:
code list IDs should use hypens not underscores please - `ott-countries-in-groups`